### ISSUE TITLE
Icon missing on Flatpak install (Ubuntu)

### ISSUE THREAD

**Issue by storcke64** [amit9838/mousam#177]
![Screenshot from 2024-11-10 19-54-11](https://github.com/user-attachments/assets/571572fe-a36b-4638-bcae-65209b0c02d7)
![Screenshot from 2024-11-10 20-00-43](https://github.com/user-attachments/assets/3143889e-21bc-40b8-8082-0f4f18b2665c)

This is happening on Ubuntu 24.04 LTS
This is not happening on Fedora.

**Comment by astro-ray** [comment]
Are you using a custom icon theme? What happens I'd you change your icon theme to default Adwaita theme?

**Comment by amit9838** [comment]
Yes it's custom icon, I believe the issue is due to the app store.

**Comment by storcke64** [comment]
I am not sure who fixed this but it appears to be fixed. Thank you kindly.
